webrender/examples/common/boilerplate_hal.rs

```diff
@@ -110,13 +111,25 @@ pub fn main_wrapper<E: Example>(
 
     let device_pixel_ratio = window.hidpi_factor();
     println!("Loading shaders...");
+
+    let rendering_api = if cfg!(feature = "dx12") {
```

Comment:
better have a local capability struct here instead of pointing to particular backends, e.g. declare a bitflags field with BLIT flag, have a run-time value (like `renderering_api` but renamed) that will only have the `BLIT` flag on Vulkan, for now.